routes/books.js

```diff
@@ -1,8 +1,62 @@
 const express = require("express");
 const generateId = require("uuid/v4");
+const authorsDB = require("../data/authors");
+const booksDB = require("../data/books");
 
 const router = express.Router();
 
 // Define your endpoints here
+router.get("/", (req, res, next) => {
+  res.json(booksDB);
+});
+
+router.post("/", (req, res, next) => {
+  const newBook = {
+    id: req.body.id,
+    author: req.body.author,
+    title: req.body.title,
+    description: req.body.description
+  };
+  booksDB.push(newBook);
+  res.json(newBook);
+});
+
+router.get("/:id", (req, res, next) => {
+  const book = booksDB.filter(el => {
+    return el.id === req.params.id;
+  });
+  res.json(book);
+});
+
+router.delete("/:id", (req, res, next) => {
+  const index = booksDB.findIndex(el => {
+    return el.id === req.params.id;
+  });
+  booksDB.slice(index, 1);
+  res.json({ message: "Job is done :joy:" });
```

Comment:
As per the instructions you should not send anything. To do so, you would write `res.sendStatus(202)`.